Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -916,6 +918,54 @@ Inspector::Protocol::ErrorStringOr<Ref<JSON::ArrayOf<String>>> InspectorCSSAgent
     return fontFamilyNames;
 }
 
+Inspector::Protocol::ErrorStringOr<Ref<JSON::ArrayOf<Inspector::Protocol::CSS::FontFace>>> InspectorCSSAgent::getCustomFonts()
+{
+    auto customFonts = JSON::ArrayOf<Inspector::Protocol::CSS::FontFace>::create();
+    auto* domAgent = m_instrumentingAgents.persistentDOMAgent();
+
+    for (auto* document : domAgent->documents()) {
+        auto iterator = document->fonts().get().createIterator(document->scriptExecutionContext());
+
+        while (auto font = iterator.next()) {
+
+            String status = ""_s;
+
+            switch (font.get()->status()) {
+            case FontFace::LoadStatus::Unloaded:
+                status = "unloaded"_s;
+                break;
+            case FontFace::LoadStatus::Loading:
+                status = "loading"_s;
+                break;
+            case FontFace::LoadStatus::Error:
+                status = "error"_s;
+                break;
+            case FontFace::LoadStatus::Loaded:
+                status = "loaded"_s;
+                break;
+            default:
+                ASSERT_NOT_REACHED();
+                break;
```

Comment:
this is unnecessary since you've already covered all the possible values of `FontFace::LoadStatus`

Reply:
I removed the assert.